### PR TITLE
feat: [CDS-46604]: account, org, home, project, module level Services/Environment

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary

project level - behind existing FF NEW_LEFT_NAVBAR_SETTINGS along with other links
account, org, home - behind FF GLOBAL_SERVICE_ENV

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/45-projects-orgs/components/ProjectsSideNav/ProjectsSideNav.tsx [hunk 1]
```diff
@@ -74,18 +75,20 @@ export default function ProjectsSideNav(): React.ReactElement {
...
+                label={getString('common.pipelineExecution')}
+                to={routes.toDeployments(projectDetailsParams)}
+              />
+              <SidebarLink
+                className={css.sidebarlink}
+                label={getString('pipelines')}
+                to={routes.toPipelines(projectDetailsParams)}
+              />
+              <SidebarLink label={getString('services')} to={routes.toServices(projectDetailsParams)} />
+              <SidebarLink label={getString('environments')} to={routes.toEnvironment(projectDetailsParams)} />
```
Comment (udayvunnam) [inline src/modules/45-projects-orgs/components/ProjectsSideNav/ProjectsSideNav.tsx:90]: All of them user NEW_LEFT_NAVBAR_SETTINGS flag

File: src/modules/75-cd/components/Environments/__test__/EnvironmentTabs.test.tsx [hunk 2]
```diff
@@ -51,13 +51,13 @@ describe('Environment Tabs', () => {
...
             class="TabNavigation--tags TabNavigation--small"
-            href="/account/undefined/undefined/orgs/undefined/projects/undefined/environment"
+            href="/account/undefined/environment"
           >
             environment
           </a>
           <a
             class="TabNavigation--tags TabNavigation--small"
-            href="/account/undefined/undefined/orgs/undefined/projects/undefined/environment-group"
+            href="/account/undefined/environment-group"
```
Comment (udayvunnam) [inline src/modules/75-cd/components/Environments/__test__/EnvironmentTabs.test.tsx:60]: Good luck identifying what changed in these snapshots tests :) 

Earlier there was only one route, now <TestWrapper> w/out any URL points to the base route.

File: src/modules/75-cd/components/Services/Services.tsx [hunk 3]
```diff
@@ -53,9 +53,13 @@ export const Services: React.FC = () => {
...
         breadcrumbs={<NGBreadcrumbs />}
-        toolbar={<TimeRangeSelector timeRange={resultTimeFilterRange?.range} setTimeRange={setTimeRange} minimal />}
+        toolbar={
+          showServicesDashboard && (
+            <TimeRangeSelector timeRange={resultTimeFilterRange?.range} setTimeRange={setTimeRange} minimal />
+          )
+        }
       />
-      {isCommunity ? (
+      {isCommunity || !showServicesDashboard ? (
```
Comment (udayvunnam) [inline src/modules/75-cd/components/Services/Services.tsx:62]: ServicesDashboard shown only at module level but not at any other level

**Comment by udayvunnam** [comment]
retrigger titlecheck

**Review by vkbansal-harness — DISMISSED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

File: src/modules/10-common/utils/routeUtils.ts [hunk 4]
```diff
@@ -233,3 +234,24 @@ export const validateReturnUrl = (url: string): boolean => {
...
+  scope: Partial<ProjectPathProps & ModulePathParams>
+  path: string
+  accountRoutePlacement?: AccountRoutePlacement
+}): string => {
+  if (orgIdentifier && projectIdentifier) {
+    const basePath = module || 'home'
+    return `/${basePath}/orgs/${orgIdentifier}/projects/${projectIdentifier}/${path}`
+  } else if (orgIdentifier) {
+    return `/settings/organizations/${orgIdentifier}/setup/resources/${path}`
+  } else if (accountRoutePlacement) {
```
Comment (chetanvekariya-harness) [inline src/modules/10-common/utils/routeUtils.ts:252]: Isn't this redundant ?

**Review by chetanvekariya-harness — APPROVED** [review]

File: src/modules/10-common/utils/routeUtils.ts (hunk 4 above)
Comment (udayvunnam) [reply src/modules/10-common/utils/routeUtils.ts:252]: No. Redundant with?

Comment (chetanvekariya-harness) [reply src/modules/10-common/utils/routeUtils.ts:252]: My bad. I misunderstood the code.

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**